README.md

```diff
@@ -21,7 +21,7 @@ as webservices.
 Please see https://ispyb.readthedocs.io.
 
 ### Requirements
-* Python 3.7, 3.8, 3.9, 3.10, 3.11
+* Python 3.7, 3.8, 3.9 (minimum), 3.10, 3.11, 3.12, 3.13 (recommended)
```

Comment:
```suggestion
* Python 3.10 (minimum), 3.11, 3.12, 3.13 (recommended)
```

Reply:
Ah yeah, good idea, I kept them as "minimums" in case someone at Diamond still used 3.7 in prod. But I think (hope?) this is not the case